Src/FluentAssertions/Specialized/ExceptionAssertions.cs

```diff
@@ -191,6 +192,54 @@ public ExceptionAssertions(IEnumerable<TException> exceptions)
             return this;
         }
 
+        private IEnumerable<Exception> GetExpectedInnerExceptions(Type innerException, string because = null,
```

Comment:
🤔 I think we should rename this method to emphasize that it's primary function is to assert something. Same for the other `Get` method.